src/app/pages/subjects/subjects.scss

```diff
@@ -0,0 +1,155 @@
+@import 'variables';
+
+$light-gray: #f5f5f5;
+$lightest-gray: #fafafa;
+$light-line: #e0e0e0;
+$text-black: rgba(0, 0, 0, 0.87);
+$text-black2: rgba(0, 0, 0, 0.7);
+$text-black3: rgba(0, 0, 0, 0.54);
+
+.text-content {
+    color: $text-black;
+    font-family: Lato;
+
+    h1 {
+        font-weight: 900;
+    }
+
+    header {
+        background: $page-top-gradient;
+        font-size: large;
+        padding: 5rem 10rem;
+        text-align: center;
+    }
+
+    .subhead {
+        font-size: 1.5rem;
+    }
+
+    .feature-block {
+        display: flex;
+        font-size: initial;
+        justify-content: space-around;
+        margin-top: 3rem;
+
+        > .blurb {
+            .title {
+                font-weight: bold;
+            }
+
+            .text {
+                max-width: 30rem;
+            }
+        }
+    }
+
+    .button-row {
+        display: flex;
+        justify-content: center;
+
+        .filter-buttons {
+            background-color: $lightest-gray;
+            color: $text-black3;
+            font-size: 120%;
+            position: relative;
+            transform: translateY(50%);
+
+            .filter-button {
+                border: thin solid $light-line;
+                cursor: pointer;
+                display: table-cell;
+                padding: 1rem 2rem;
+
+                &.selected {
+                    color: $text-black;
+                }
+            }
+        }
+    }
+
+    .book-viewer {
+        background-color: $light-gray;
+        padding: 5rem 10rem;
+
+        .subject {
+            color: $text-black2;
+            font-size: 120%;
```

Comment:
Your comments are appreciated, and I'll put together a branch to fix the
CSS issues. But you're going to drive yourself crazy reviewing the HTML and
content. That's just scaffolding. I'm expecting the design guys to produce
the real HTML and styling. Maybe that's a misplaced expectation on my part?

On font size percents: Sometimes I just want to indicate that the font
should be a bit bigger than it otherwise would be. Do you consider that a
bad practice? I went through and changed most of them to equivalent rems
(the one you commented on I apparently overlooked), but in my mind, it's
actually correct to say that I just want them bigger. your view?


On Wed, Feb 10, 2016 at 10:43 PM, Derek Kent <notifications@github.com>
wrote:

> In src/app/pages/subjects/subjects.scss
> <https://github.com/openstax/os-webview/pull/21#discussion_r52564070>:
>
> > +                padding: 1rem 2rem;
> > +
> > +                &.selected {
> > +                    color: $text-black;
> > +                }
> > +            }
> > +        }
> > +    }
> > +
> > +    .book-viewer {
> > +        background-color: $light-gray;
> > +        padding: 5rem 10rem;
> > +
> > +        .subject {
> > +            color: $text-black2;
> > +            font-size: 120%;
>
> Should this be rem?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openstax/os-webview/pull/21/files#r52564070>.
>



-- 
Roy Johnson
Javascript Developer
OpenStaxx